spatial/curve/hilbert_test.go

```diff
@@ -0,0 +1,346 @@
+package curve
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+
+	"golang.org/x/exp/rand"
+)
+
```

Comment:
Please place the `func Test...` first, followed by the `func Bench...` with the callees after their last dependent.